### ISSUE TITLE
The Close button

### ISSUE THREAD

**Issue by notecode** [euvl/vue-js-modal#16]
First, I like your work very. 

But I found a close button on the top-right corner is better. What is your thought about this?
I am willing to add it based on your code, but I don't want to modify your code directly, so I want to compose your component. But It's pity that I cannot find a proper way. (I have learnt vuejs just a few days)

**Comment by euvl** [comment]
Hey @notecode thanks for the kind words and using plugin 😄 

I did not plan to add the close button yet for a couple of reasons. First, i didnt have a use-case :D. Second, to have a nice "X" symbol, I will have to use some external library (FontAwesome or whatever) or create an svg asset. I was trying to keep plugin simple, without any external deps. Third, I thought that "click-outside-to-close' is pretty natural ux pattern nowadays.

But you are right saying that there should be some way to make it all work, I will think what we could do and also if you will have any ideas please let me know or feel free to create a pull request 😄 

Thanks

**Comment by notecode** [comment]
Hey @euvl 
I've tried like this: Add these lines in your Modal.vue

in the html
`<span v-if="x" @click="toggle(false)" :class="xClass">x</span>`


in the props

   x: {
        type: Boolean,
        default: false,
      },  
      xClasses: {
        type: [String, Array],
        default: '', 
      }   

in the computed:

    xClass () {
        return ['x', this.xClasses]
      }

in the style

 .x {
    display: inline-block;
    position: absolute;
    right: 9px;
    top: 2px;
    color: #333333;
    font-size: 20px;
    cursor: pointer;
  }
  .x:hover {
    color: gray;
  }

My thought:
1. Yeah, the 'click-outside-to-close' is common now, but some 'old' UI designers still like the x button, like the one in my team. So, give the choice to the user.
2. Just use a simple and ugly 'x' button via char 'x',  If the user need a pretty one, he will customize by himself.

I want to make a pull request, but failed. I've need contribute my code to the open-source world yet.  (I thought I should make a new branch first, so I made one in my local, but I can not push it to github) So, I copy my code here to show you.
